src/Payments.sol

```diff
@@ -390,19 +390,47 @@ contract Payments is
             rail.from
         ][rail.operator];
 
-        // Ensure account lockup is fully settled up to the current epoch
-        uint256 lockupSettledUpto = settleAccountLockup(payer);
         require(
-            lockupSettledUpto == block.number,
-            "cannot modify active rail lockup: client funds insufficient for current account lockup settlement"
+            !isRailInDebt(rail, payer),
+            "cannot modify rail lockup: rail is in debt"
         );
```

Comment:
Remove this? We're checking this below and we can't check it until we've settled lockup.

Reply:
Done.